src/pynws/forecast_units.py

```diff
@@ -0,0 +1,13 @@
+import sys
+
+if sys.version_info >= (3, 11):
+    from enum import StrEnum
+else:
+    from .backports.enum import StrEnum
+
+
+class NwsForecastUnits(StrEnum):
```

Comment:
Should we move this to `const.py` where other values are stored? Or will we prefer to make them separate in the future?

Reply:
Also, not sure this needs the `Nws` prefix as that isn't done elsewhere.

Reply:
I agree with moving to const.py.

We have a bit of mixed bag on naming currently.  Most constants do not have a Nws prefix, although the error does.  In my experience most python packages do not prefix the package, or related, name on variables inside their namespace.  I would vote for no prefix to be most consistent with the rest of the package.

Reply:
My idea behind the separate file was because this class would be "public" vs the more "private" values that are stored in the `const.py`, but this makes sense too.